modules/administration-guide/pages/enabling-fuse-for-all-workspaces.adoc

```diff
@@ -7,6 +7,11 @@
 [id="enabling-fuse-overlayfs-for-all-workspaces"]
 = Enabling fuse-overlayfs for all workspaces
 
+For Podman 5.x, the `/home/user/.config` must be owned by the current user for Podman to function correctly.
+The `storage.conf` file for Podman is typically stored in this folder.
+This document explains how to configure the container entrypoint script for the workspace so that fuse-overlayfs is being used.
```

Comment:
after configuration, is fuse-overlayfs used by the workspace with the configured script? the wording is a bit unclear, especially with "so that fuse-overlayfs is being used."

Reply:
The script is configured for the container image entrypoint. After that, a new container image is built, and whatever workspace uses that container image, will be using fuse-overlay fs for podman